using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Week1Task
{
    class tryCatch
    {
        /*public static void Main(string[] args)
        {
            try
            {
                int result = Divide(10, 0);
                Console.WriteLine("Result: " + result);
            }
            catch (DivideByZeroException e)
            {
                Console.WriteLine("An divide by zero exception occurred: " + e.Message);
            }
            catch (Exception e)
            {
                Console.WriteLine("An exception occurred: " + e.Message);
            }


        }

        private static int Divide(int a, int b)
        {
            if (b == 0)
            {
                throw new DivideByZeroException("Cannot divide by zero.");
            }
            return a / b;
        }*/
    }
}
